### ISSUE TITLE
uploaded logo in Appearance admin section does not show up after upload

### ISSUE THREAD

**Issue by individual-it** [owncloud/ocis#6589]
## Describe the bug

in the  Appearance admin section its possible to upload an own logo, this new logo only shows after a force reload of the browser

## Steps to reproduce

1. login as admin
2. go to the Administration settings -> General
3. upload a logo
4. wait for the automatic reload

## Expected behavior
The new logo should be visible

## Actual behavior
the old logo is visible
only after a page reload with pressed "Shift" key the new logo shows up

## Setup

Please describe how you started the server and provide a list of relevant environment variables or configuration files.

<details>
<p>

```console
docker run --name ocis -p 9200:9200 --mount type=bind,source=/mnt/HC_Volume_32687865/ocis-config,target=/etc/ocis --mount type=bind,source=/mnt/HC_Volume_32687865/ocis-data,target=/var/lib/ocis  --mount type=bind,source=/mnt/HC_Volume_32687865/ocis-logs,target=/var/log/ocis -e OCIS_INSECURE=true -e PROXY_HTTP_ADDR=0.0.0.0:9200 -e OCIS_URL=https://ocis.in-nepal.de -e PROXY_TLS=false -e OCIS_LOG_FILE=/var/log/ocis/ocis.log -d --restart=always owncloud/ocis:latest
```

</p>
</details>

## Additional context
```
Version: 3.1.0-next.2+2c0fb9910
Web client version: 7.0.1

**Comment by micbar** [comment]
Duplicate of #6122
